<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CryptoSignDesktop MIREA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/MIREA_Gerb_Colour.png</normaloff>:/resources/MIREA_Gerb_Colour.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_files">
      <property name="title">
       <string>Файлы</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="my_tableWidget" name="tableWidget_files">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_filesAdd">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_filesRemove">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_sign">
      <property name="title">
       <string>Подпись</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_createSign">
         <property name="title">
          <string>Создание подписи</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="8" column="1" colspan="2">
           <widget class="QPushButton" name="pushButton_createSign">
            <property name="text">
             <string>Подписать</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_signType">
            <property name="text">
             <string>Тип подписи</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit_outputDir">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_signFormat">
            <property name="text">
             <string>Формат</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_insertType">
            <property name="text">
             <string>Тип вставки</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="3">
           <widget class="QCheckBox" name="checkBox_drawLogo">
            <property name="text">
             <string>Рисовать герб</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QToolButton" name="toolButton_signInsertTypeInfo">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/info.png</normaloff>:/resources/info.png</iconset>
            </property>
           </widget>
          </item>
          <item row="7" column="3">
           <widget class="QToolButton" name="toolButton_choseOutputDir">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/folder.png</normaloff>:/resources/folder.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_drawLogo">
            <property name="text">
             <string>Герб:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QComboBox" name="comboBox_signType">
            <item>
             <property name="text">
              <string>Отсоединённая</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Присоединённая</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Нет (демо)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_outputDir">
            <property name="text">
             <string>Папка вывода</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QComboBox" name="comboBox_insertType">
            <item>
             <property name="text">
              <string>В конец документа</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>По тэгу</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QComboBox" name="comboBox_signFormat">
            <item>
             <property name="text">
              <string>DER</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>base64</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_choseCertificate">
         <property name="title">
          <string>Выбор сертификата</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="2">
           <widget class="QRadioButton" name="radioButton_displayName">
            <property name="text">
             <string>Владелец</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_signName">
            <property name="text">
             <string>Название: </string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="2">
           <widget class="QLabel" name="label_signOwnerValue">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QToolButton" name="toolButton_searchCertificate">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/search_green.png</normaloff>:/resources/search_green.png</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_signOwner">
            <property name="text">
             <string>Владелец:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_signCertificate">
            <property name="text">
             <string>Сертификат</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QComboBox" name="comboBox_certificates"/>
          </item>
          <item row="8" column="2" colspan="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="2" colspan="2">
           <widget class="QLabel" name="label_signCertificateValue">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_signConfirmDates">
            <property name="text">
             <string>Действителен: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QRadioButton" name="radioButton_displayLabel">
            <property name="text">
             <string>Название</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2" colspan="2">
           <widget class="QLabel" name="label_signConfirmDatesValue">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2" colspan="2">
           <widget class="QLabel" name="label_signNameValue">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_signChoseVisible">
            <property name="text">
             <string>Отображение:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_signChose">
            <property name="text">
             <string>Сертификат</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="4">
           <widget class="previewWidget" name="widget_preview" native="true">
            <property name="minimumSize">
             <size>
              <width>384</width>
              <height>160</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>384</width>
              <height>160</height>
             </size>
            </property>
            <widget class="QWidget" name="gridLayoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>361</width>
               <height>141</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>ДОКУМЕНТ ПОДПИСАН</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_priviewCertificate">
                <property name="text">
                 <string>Сертификат:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>ЭЛЕКТРОННОЙ ПОДПИСЬЮ</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_priviewOwner">
                <property name="text">
                 <string>Владелец:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_priviewDateFromTo">
                <property name="text">
                 <string>Действителен:</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>my_tableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">my_tablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>previewWidget</class>
   <extends>QWidget</extends>
   <header location="global">previewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
